lwt/nbd_lwt_unix.ml

```diff
@@ -21,13 +21,9 @@ type tls_role =
   | TlsClient of Ssl.context
   | TlsServer of Ssl.context
 
-let debug_io = ref false
-
-(* TODO move this function into a library used by this and other executables *)
+(* TODO move this function into a library used by both this and vhd-tool. *)
 (* Also the read/write/close functions that are inside tls_channel_of_fd *)
-let io_complete name offset op fd buffer =
-  if !debug_io
-  then Printf.fprintf stderr "%s offset=%s length=%d\n%!" name (match offset with Some x -> Int64.to_string x | None -> "None") (Cstruct.len buffer);
+let io_complete op fd buffer =
```

Comment:
We could also give some concrete examples in the ocamldoc, like here: https://github.com/mirage/ocaml-cstruct/blob/master/lwt/lwt_cstruct.mli#L30